package.json

```diff
@@ -44,8 +44,8 @@
     "@types/selenium-webdriver": "^4.0.0",
     "@types/wavesurfer.js": "^3.3.1",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
-    "@typescript-eslint/parser": "^4.0.0",
-    "ajv": "^6.10.0",
+    "@typescript-eslint/parser": "4.13.0",
```

Comment:
There are some errors in newer version of the parser that lead to false errors reported.